spring-multirabbit/src/main/java/org/springframework/amqp/rabbit/annotation/ExtendedMultiRabbitListenerAnnotationBeanPostProcessor.java

```diff
@@ -23,9 +26,15 @@
  * @author Wander Costa
  * @see RabbitListenerAnnotationBeanPostProcessor
  */
+@Component
```

Comment:
`ExtendedMultiRabbitListenerAnnotationBeanPostProcessor` is being registered in the scope of `MultiRabbitBootstrapConfiguration` if multi is enabled. It should not be a bean always available (for the sake of the compatibility with non-multirabbit projects importing/inhering it).
https://github.com/freenowtech/spring-multirabbit/blob/main/spring-multirabbit/src/main/java/org/springframework/amqp/rabbit/annotation/MultiRabbitBootstrapConfiguration.java#L37

Did you test without it being a component?

Reply:
Hi, Wander. Yes, that is correct, `@Component` is not required in this context. Removed 👍🏻